core/modules/node/config/views.view.recent_content.json

```diff
@@ -658,7 +659,8 @@
                     "more_link": 0,
                     "path_override": 0,
                     "fields_override": 0
-                }
+                },
+                "link_url": "/admin/content"
```

Comment:
I think these should be `admin/content`, and Backdrop will run that through `url()` to expand to the base path.